### DISCUSSION TITLE
Which Arch distro are you guys using as a base?

### DISCUSSION THREAD

**Discussion by 7thripr** [prasanthrangan/hyprdots#52] [Q&A]
I used EndeavourOS but hyprland wont load after login, so which base are you guys using straight up base arch iso?

**Comment by prasanthrangan** [comment]
It should work on EndeavourOS as well, is this the nvidia issue?

**Reply by 7thripr** [reply]
i think i relaced hyprland with hyprland-nvidia and it was working fine took some nvidia specific options from other hyprland scripts and was able to get it to work.

**Comment by DiegoSof2** [comment]
I'm using manjaro

**Comment by Gedankenn** [comment]
arch

**Comment by MostafaMohamed2002** [comment]
Manjaro

**Comment by jpx40** [comment]
Garuda
